src/Image/Exif.php

```diff
@@ -242,10 +198,8 @@ protected function parse()
 
 	/**
 	 * Return the timestamp when the picture has been taken
-	 *
-	 * @return string|int
 	 */
-	protected function parseTimestamp()
+	protected function parseTimestamp(): string|int
```

Comment:
The `$timestamp` prop supports `string|null`.

The type hints of this method and the prop taken together, it seems like the `$timestamp` can only ever be `string` in the end.